app/src/main/java/com/techcourse/dao/UserDao.java

```diff
@@ -1,121 +1,54 @@
 package com.techcourse.dao;
 
 import com.techcourse.domain.User;
+import java.util.List;
+import javax.sql.DataSource;
 import nextstep.jdbc.JdbcTemplate;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-
-import javax.sql.DataSource;
-import java.sql.Connection;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.util.List;
+import org.springframework.jdbc.core.RowMapper;
 
 public class UserDao {
 
     private static final Logger log = LoggerFactory.getLogger(UserDao.class);
 
-    private final DataSource dataSource;
+    private final JdbcTemplate jdbcTemplate;
+    private final RowMapper<User> userRowMapper = (rs, rowNum) -> new User(
+            rs.getLong(1),
+            rs.getString(2),
+            rs.getString(3),
+            rs.getString(4));
```

Comment:
c: 순서보다 컬럼명이 더 명확하다고 생각했는데 에덴은 어떻게 생각하시나요?
개인적으로 getString이 3개가 있어서 구분이 잘안되는 것 같아요! 
```suggestion
    private final RowMapper<User> userRowMapper = (rs, rowNum) -> new User(
            rs.getLong("id"),
            rs.getString("account"),
            rs.getString("password"),
            rs.getString("email"));
```

Reply:
앗 다른 구현에만 집중하다가 이 부분에 대해서는 미처 신경쓰지 못했네요 😢 
가독성이 뛰어난 제로의 의견을 반영하여 수정하였습니다!!